pyproject.toml

```diff
@@ -17,9 +17,10 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Topic :: Scientific/Engineering :: Artificial Intelligence for Inference Accelerator",
 ]
+requires-python = "==3.8.*"
 dependencies = [
     "transformers==4.41.2",
-    "torch@https://download.pytorch.org/whl/cpu/torch-2.0.0%2Bcpu-cp38-cp38-linux_x86_64.whl#sha256=354f281351cddb590990089eced60f866726415f7b287db5105514aa3c5f71ca",
+    "torch==2.0.1",
```

Comment:
better to have `cpu `version only? lots of unnecessary package uninstallations due to gpu version.
Maybe we can do `torch==2.0.0+cpu @https://download.pytorch.org/whl/cpu`?

Reply:
I tried different versions like `torch==2.0.1+cpu` and with index url etc. Only thing that works without erroring is `torch@https://download.pytorch.org/whl/cpu/torch-2.0.1%2Bcpu-cp38-cp38-linux_x86_64.whl#sha256=8046f49deae5a3d219b9f6059a1f478ae321f232e660249355a8bf6dcaa810c1`, I'll add that for now.